Enhance CONTRIBUTING.md with testing instructions (#657)
zowe/cics-for-zowe-client: CONTRIBUTING.md

- **SDK, CLI package, and vsce-api packages**: npm tarballs
- **VSCE package**: .vsix file for VS Code extension installation

### Testing if you don't have a CICS install

Our tests use wiremock to fake up a CICS region and test against. You can test against these using our set of test:e2e scripts (search in [packages/vsce/package.json](https://github.com/zowe/cics-for-zowe-client/blob/main/packages/vsce/package.json)).

> You need a docker setup locally - on the Mac, we use colima.

Reviewer: ```suggestion
You need Docker setup locally - on macOS, we use Colima as the Docker runtime.
```

Start colima if necessary:
```
colima start
```
From within `packages/vsce`...
Tear down any existing running e2e tests:
```
npm run test:e2e:teardown

> ```

Reviewer 2: after tear down, we can use cd packages/vsce
	npm run test:e2e:setup and then npm run test:e2e:dev:interactive as you have mentioned below, also we can mention if you want to stop colima use colima stop.

I guess npm run test:e2e:teardown should be outside of vsce package.

Reviewer: `setup` doesn't need to be run manually, it's included in the `dev:interactive` script 👍🏼
